Add Links aparecem no front

HeullerRamos opened the pull request.

HeullerRamos committed:
Add função de ligação entre User e Servidor

HeullerRamos committed:
Add função de ligação entre Servidor e Professor

HeullerRamos committed:
Alteração na função edit para puxar de forma correta todos os dados do banco

MarcosMMarques reviewed (changes requested):
Links on the Update Profile page are still not displaying correctly.

HeullerRamos committed:
add forelse para buscar os links do professor no banco

HeullerRamos committed:
Migrações para alteração dos links do professor

HeullerRamos committed:
model p ligação entre link e curriculo_professor

HeullerRamos committed:
Update CurriculoProfessor.php

Alteração realizada para condizer com a nova estrutura do banco, remoção da coluna link em curriculo professor e criação da tabela links

HeullerRamos committed:
nova forma de salvar links na tabela correspondente

HeullerRamos committed:
ligação entre os curriculos  e os professores

HeullerRamos committed:
Alteração para busca dos links nos curriculos

HeullerRamos committed:
Alteração para busca dos links nos curriculos

HeullerRamos committed:
migração para nova exclusão de usuário

HeullerRamos committed:
import softdelete para nova exclusão de usuario

HeullerRamos committed:
atualização no gerador do curriculo/link professor

HeullerRamos committed:
adição e exclusando de links

HeullerRamos committed:
alteração na forma criar e apagar os links

HeullerRamos committed:
add rota para exclusão dos links

HeullerRamos committed:
Update web.php

HeullerRamos committed:
add controller para os links

HeullerRamos committed:
Mensagens de excessão para os links

HeullerRamos committed:
nova forma de salvar os novos links

Alteração realizadas para trabalhar em conjunto com as excessões criadas.

HeullerRamos committed:
mensagens de excessão para troca de senha

Felkng committed:
Merge branch 'dev' into perfil

Felkng commented:
ball show d+++
